spec/requests/api/logging_spec.rb

```diff
@@ -0,0 +1,51 @@
+#
+# REST API Logging Tests
+#
+describe ApiController do
+  describe "Successful Requests logging" do
+    LOG_REQUESTS = {
+      "API Request"    => nil,
+      "Authentication" => nil,
+      "Authorization"  => nil,
+      "Request"        => nil,
+      "Parameters"     => nil,
+      "Response"       => nil
+    }.freeze
+
+    def expect_log_requests(expectations)
+      expectations.each do |category, expectation|
+        expect_any_instance_of(ApiController).to receive(:log_request)
```

Comment:
thought about that but that just takes the resolved string, testing :log_request gives us better control so we can check for individual hash entries and easier to extend the logging specs.